lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -2,6 +2,7 @@
 require 'json'
 require 'kafo_parsers/doc_parser'
 require 'kafo_parsers/param_doc_parser'
+require 'open3'
```

Comment:
This is perhaps a Python dev thing, but I always import built ins before things from my own package.